app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
We were seeing it in production. This only applies to the meta tags. If dateCreated isn't there, it's not really Preprint's job to figure out what the desired behavior is. We the API contract dictates that it will always be there. Having a bad date for a scraper is better than throwing an error for the entire page. I was planning on verifying that #239 works once it's up on staging. We'll probably have merge conflicts.